lib/state_machines/audit_trail/backend/active_record.rb

```diff
@@ -1,13 +1,11 @@
 require 'state_machines-activerecord'
 
 class StateMachines::AuditTrail::Backend::ActiveRecord < StateMachines::AuditTrail::Backend
-  attr_accessor :context
-
-  def initialize(transition_class, owner_class, context = nil)
+  def initialize(transition_class, owner_class, options = {})
+    super
     @association = transition_class.to_s.tableize.split('/').last.to_sym
-    super transition_class, owner_class
-    self.context = context # FIXME: actually not sure why we need to do this, but tests fail otherwise. Something with super's Struct?
```

Comment:
Note that `super` here was being called without `context` as a third arg to save in the struct; this is why line 9 was needed.